src/main/msp/msp_protocol.h

```diff
@@ -319,6 +319,7 @@
 #define MSP_SERVO_MIX_RULES      241    //out message         Returns servo mixer configuration
 #define MSP_SET_SERVO_MIX_RULE   242    //in message          Sets servo mixer configuration
 #define MSP_SET_4WAY_IF          245    //in message          Sets 4way interface
+#define MSP_SET_RTC              246    //in message          Sets the RTC clock - TODO: Make sure this code is not used by other forks
```

Comment:
Let's not touch MSPv1 messages. Please move RTC MSP API to `msp_protocol_v2_inav.h` header and change the ID according to https://github.com/iNavFlight/inav/wiki/MSP-V2#msp-v2-message-catalogue (change to range 0x1000 - 0x1FFF)

Reply:
I'm not sure about this. `MSP_SET_RTC` seems a valid v1 message, since it could be used by the Taranis to set the time. As such the code could be shared with betaflight. Currently neither Taranis nor Betaflight understand MSPv2.

Reply:
Good point. Let's keep it MSPv1 then for compatibility.

Reply:
MSPv1 it is then. Is 246 OK to use?

Reply:
Any unused value should do. Value of 246 seems to be unused across INAV, BF and CF